app/institutions/dashboard/route.ts

```diff
@@ -0,0 +1,21 @@
+import { action } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Route from '@ember/routing/route';
+import DS from 'ember-data';
+
+import Analytics from 'ember-osf-web/services/analytics';
+
+export default class InstitutionsDashboardRoute extends Route {
+    @service analytics!: Analytics;
+    @service store!: DS.Store;
+
+    // eslint-disable-next-line camelcase
+    model(params: { institution_id: string }) {
+        return this.store.findRecord('institution', params.institution_id);
```

Comment:
I know this is a placeholder route, but I just wanted to note that we've generally avoided returning promises in model hooks so that page rendering is not blocked. Instead, take a look at
https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/meetings/detail/route.ts
for an example of how we use an ember-concurrency task to handle the data loading and return an instance of it in an object as the model. The whole `meetings/detail` route and related components are actually very similar in structure to the institutional dashboard.

Reply:
Just wanted to check if you wanted this before merging? Was planning on making this update with the next route ticket, but can do it now too if you want. 😄

Reply:
Nah, don't worry about it now.